From 13ee3e016490e74868b64e3a07dcccf9feafebdf Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 30 Nov 2021 05:59:06 -0800
Subject: [PATCH] configure.ac: drop 2nd AM_INIT_AUTOMAKE

* automake-1.16.5 introduced in oe-core:
  https://git.openembedded.org/openembedded-core/commit/?id=851167b3a41b1728407d331c1666827fb730daa1
  doesn't like this after:
  http://git.savannah.gnu.org/cgit/automake.git/commit/?id=f4a3a70f69e1dbccb6578f39ef47835098a04624

  and do_configure fails with:
  configure.ac:38: error: AM_INIT_AUTOMAKE expanded multiple times

  There is no point in upstreaming this, because singularity-2.3.1 is very old and
  whole autotools support is removed in version 3 (currently 3.8.5) with:
  https://github.com/hpcng/singularity/commit/a06e3d13a822080d7a9bc55085ee1bb32026a96e

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 configure.ac | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8ffa5ab32..a5a35c43a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -35,7 +35,6 @@ AC_GNU_SOURCE
 AC_PROG_INSTALL
 AC_PROG_LIBTOOL
 AC_PROG_CC
-AM_INIT_AUTOMAKE
 AM_PROG_CC_C_O
 AC_ENABLE_SHARED
 AC_PROG_LIBTOOL(libtool)
